src/main/java/net/fabricmc/loader/api/EntrypointException.java

```diff
@@ -37,6 +39,14 @@ public EntrypointException(String key, String causingMod, Throwable cause) {
 		this.key = key;
 	}
 
+	/**
+	 * @deprecated For internal use only, to be removed!
+	 */
+	public EntrypointException(String key, ModMetadata causingMod, Throwable cause) {
+		super("Exception while loading entries for entrypoint '" + key + "' provided by '" + causingMod.getName() + "' (" + causingMod.getId() + ")", cause);
+		this.key = key;
```

Comment:
key should have a better name

Reply:
not gonna add more PR noise

Reply:
It is the key for entrypoint, like `main` for `ModInitializer` etc